Ginger/GingerCoreCommon/ReporterLib/UserMsgsPool.cs

```diff
@@ -222,7 +222,7 @@ public static void LoadUserMsgsPool()
             Reporter.UserMsgsPool.Add(eUserMsgKey.FailedToAddItemToSharedRepository, new UserMsg(eUserMsgType.ERROR, "Add Item to Repository", "Failed to add the item '{0}' to shared repository." + Environment.NewLine + Environment.NewLine + "Error Details: {1}.", eUserMsgOption.OK, eUserMsgSelection.None));
             Reporter.UserMsgsPool.Add(eUserMsgKey.AskIfWantsToUpdateRepoItemInstances, new UserMsg(eUserMsgType.WARN, "Update Repository Item Usages", "The item '{0}' has {1} instances." + Environment.NewLine + Environment.NewLine + "Do you want to review them and select which one to get updated as well?", eUserMsgOption.YesNo, eUserMsgSelection.No));
             Reporter.UserMsgsPool.Add(eUserMsgKey.AskIfWantsToChangeeRepoItem, new UserMsg(eUserMsgType.WARN, "Change Repository Item", "The item '{0}' is been used in {1} places." + Environment.NewLine + Environment.NewLine + "Are you sure you want to {2} it?" + Environment.NewLine + Environment.NewLine + "Note: Anyway the changes won't affect the linked instances of this item", eUserMsgOption.YesNo, eUserMsgSelection.No));
-            Reporter.UserMsgsPool.Add(eUserMsgKey.AskIfWantsToChangeeRepoItem2, new UserMsg(eUserMsgType.WARN, "Change Repository Item", "The item '{0}' may be used in many places." + Environment.NewLine + Environment.NewLine + "Are you sure you want to {1} it?" + Environment.NewLine + Environment.NewLine + "Note: Anyway the changes won't affect the linked instances of this item", eUserMsgOption.YesNo, eUserMsgSelection.No));
+            Reporter.UserMsgsPool.Add(eUserMsgKey.AskIfWantsToChangeeRepoItem2, new UserMsg(eUserMsgType.WARN, "Change Repository Item", "The item '{0}' may be used in many places." + Environment.NewLine + Environment.NewLine + "Are you sure you want to {1} it?" + Environment.NewLine + Environment.NewLine + "Note: Changes will reflect in linked instances of this item", eUserMsgOption.YesNo, eUserMsgSelection.No));
```

Comment:
Rename the message key AskIfWantsToChangeeRepoItem2 to AskIfWantsToChangeLinkedRepoItem

Also the message, let's keep it simple like below

"Modifying linked item will auto update all linked instances."  
 Are you sure you want to {1} it?"

Reply:
chaneged enum name and updated the message